packages/amplify-gen1-codegen-auth-adapter/src/auth_render_adapter.ts

```diff
@@ -16,6 +16,7 @@ import {
   LoginOptions,
   Scope,
 } from '@aws-amplify/amplify-gen2-codegen';
+import { AttributeMappingRule } from '@aws-amplify/amplify-gen2-codegen/src/auth/source_builder';
```

Comment:
Is it intentional that changes in `auth-adapter` are part of `function codegen` ?
I.e. this doesn't match PR description.

Reply:
Yes, the changes are intentional. I updated the description